### DISCUSSION TITLE
Can I build on MacOS with a lower Xcode version?

### DISCUSSION THREAD

**Discussion by orisonchan** [AppFlowy-IO/AppFlowy#187] [Technical help wanted] [answered]
Building will use Flutter.
Flutter recommends a minimum Xcode version of 13.0.0.
But Xcode 13.0 seems to support at least Big Sur 11.3.
I think it's too high. Big Sur and Monterey have some bugs so that my colleague and I still use Catalina...

**Comment by annieappflowy** [comment]
@appflowy

**Reply by orisonchan** [reply]
Thank you!

**Comment by appflowy** [comment, marked as answer]
@orisonchan You can build with a lower Xcode version. If you have any issues, please me on discord.

**Reply by orisonchan** [reply]
Thank you for reply! Got it!
